wraith/bin/wraith.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+read -p "Specify the port number that your local dev environment is running on: " port
+
+port=${port:-5000}
```

Comment:
On reflection, why don't we just hard code port 5000 in the config file and dispense with all the trickery? I think it's fair to assume this project is bound to 5000 locally, since that's what foreman starts you up on.